R-packages/evalcast/R/target.R

```diff
@@ -51,7 +51,7 @@ get_target_period <- function(forecast_date, incidence_period, ahead) {
 #'
 #' @importFrom rlang .data
 #' @importFrom tibble enframe
-#' @importFrom dplyr mutate
+#' @importFrom dplyr mutate left_join
```

Comment:
Thanks.  We should also add `rename` and `bind_rows` to this line.

Reply:
And `group_by` and `summarize`

Reply:
Ok sure (though to be clear, these are already in the NAMESPACE, so adding them here wouldn't do anything).